benchmark.sh

```diff
@@ -0,0 +1,10 @@
+if [ ! -f ./interpreter-c ] && [ $# -lt 1 ]; then
+    echo "[ERROR] ./interpreter-c not found: Need an argument. Usage: ./benchmark.sh [PYTHONPATH]"
+else
+    if [ ! -f ./interpreter-c ]; then
+        $1 -O0 interpreter.py
+    fi
+    make clean
+    make
+    ./multitime -n5 ./interpreter-c fibonacci.bc
```

Comment:
On second thoughts, if you put the multitime part into a Make target, make can check the interpreter is built using dependencies.